server/src/main/java/com/datastrato/gravitino/server/web/rest/RoleOperations.java

```diff
@@ -33,12 +44,54 @@
 public class RoleOperations {
   private static final Logger LOG = LoggerFactory.getLogger(RoleOperations.class);
 
+  private static final String PRIVILEGE_ERROR_MSG = "%s shouldn't bind the privilege %s";
+  private static final String UNSUPPORTED_ERROR_MSG = "%s uses an unsupported catalog type";
+  private static final List<Privilege> CATALOG_PRIVILEGES =
```

Comment:
As you mentioned before that catalog will have the schema privileges like create schema, how do you achieve this in this PR?

Reply:
You can see the comment of `checkSecurableObjectPrivileges`